examples/if_ls_glob.bats

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bats
+
+TEST_DIR="test_dir"
+
+setup() {
+  mkdir "${TEST_DIR}"
+  cd "${TEST_DIR}"
+
+  bats() {
+    true
+  }
+}
+
+teardown() {
+  cd ..
+  rm -rf "${TEST_DIR}"
+}
+
+main() {
+  if ls *.bats >/dev/null 2>&1; then
+    bats -t *.bats
+  fi
+}
+
+@test "it works when there are no .bats files" {
+  run main
+  [ "${status}" -eq 0 ]
+}
+
+@test "it works when there is one .bats file" {
+  touch a.bats
+
+  run main
+  [ "${status}" -eq 0 ]
+}
+
+@test "it works when there are two .bats file" {
+  touch {a,b}.bats
+
+  run main
+  [ "${status}" -eq 0 ]
+}
+
+@test "it works when there are three .bats file" {
+  touch {a,b,c}.bats
```

Comment:
is it possible to not use `.bats` extension as for the beginners it is not clear why `.bats` was used specifically. Using `.txt` would be more straightforward.